source/basic.tex

```diff
@@ -3451,6 +3451,9 @@
   the pointer is used as the operand of a
   \keyword{dynamic_cast}\iref{expr.dynamic.cast}.
 \end{itemize}
+\begin{note}
+For an object with a non-trivial constructor or destructor, see also~\ref{class.cdtor}.
+\end{note}
```

Comment:
We already have this cross-reference in line 3425:

> For an object under construction or destruction, see~\ref{class.cdtor}.

In general, [class.cdtor] describes behavior for objects under construction or destruction, and [basic.life] describes behavior for objects that are (entirely) out-of-lifetime.

Reply:
And yes, "object" might be a subobject.